MAESTRO/scATAC_Genescore.py

```diff
@@ -72,7 +72,7 @@ def ExtractGeneInfo(gene_bed):
 
     bed = pd.read_csv(gene_bed, sep="\t", header=0, index_col=False)
     bed['transcript'] = list(map(lambda x: x.strip().split(".")[0], bed['name'].tolist()))
-    bed["chrom"] = list(map(lambda x:"chr%s"%x, bed["chrom"]))
```

Comment:
for https://raw.githubusercontent.com/liulab-dfci/MAESTRO/master/MAESTRO/annotations/GRCm38_refgenes.txt the chrom column has "chr", for https://raw.githubusercontent.com/liulab-dfci/MAESTRO/master/MAESTRO/annotations/GRCh38_refgenes.txt the chrom column does not have "chr"

@DongqingSun96 can you make sure the script works for both human and mouse?

Reply:
Just styling: Now in Python3, this expression is more recommended: `[f"chr{x}" for x in bed["chrom"]]`

Reply:
list comprehension is more pythonic. I think map() is going to retire.

Reply:
@crazyhottommy Here you list are the old `_refgenes.txt` files on the master branch. I have update these files for both GRCh38 and GRCm38. Please check this commit [https://github.com/liulab-dfci/MAESTRO/pull/65/commits/f538adb5fa181681054d28b4cbe1e391b7784a0a](https://github.com/liulab-dfci/MAESTRO/pull/65/commits/f538adb5fa181681054d28b4cbe1e391b7784a0a) on the current branch  for the updated ones.

Reply:
@taoliu Thanks, Tao. I have made some modifications as you suggested. Please check the new PR [https://github.com/liulab-dfci/MAESTRO/pull/67](https://github.com/liulab-dfci/MAESTRO/pull/67).